drivers/sdhc/sdhc_esp32.h

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define SDMMC_FREQ_DEFAULT   20000 /*!< SD/MMC Default speed (limited by clock divider) */
+#define SDMMC_FREQ_HIGHSPEED 40000 /*!< SD High speed (limited by clock divider) */
+#define SDMMC_FREQ_PROBING   400   /*!< SD/MMC probing speed */
+#define SDMMC_FREQ_52M       52000 /*!< MMC 52MHz speed */
+#define SDMMC_FREQ_26M       26000 /*!< MMC 26MHz speed */
+
+/* SD commands remapping */
+#define MMC_WRITE_BLOCK_MULTIPLE SD_WRITE_MULTIPLE_BLOCK
```

Comment:
Is there a specific reason we need these defines, and can't use `SD_WRITE_MULTIPLE_BLOCK` in the driver?